apis/docdb/v1alpha1/zz_db_instance.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// DBInstanceParameters defines the desired state of DBInstance
+type DBInstanceParameters struct {
+	// Region is which region the DBInstance will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// Indicates that minor engine upgrades are applied automatically to the instance
+	// during the maintenance window.
+	//
+	// Default: true
+	AutoMinorVersionUpgrade *bool `json:"autoMinorVersionUpgrade,omitempty"`
+	// The Amazon EC2 Availability Zone that the instance is created in.
+	//
+	// Default: A random, system-chosen Availability Zone in the endpoint's AWS
+	// Region.
+	//
+	// Example: us-east-1d
+	AvailabilityZone *string `json:"availabilityZone,omitempty"`
+	// The identifier of the cluster that the instance will belong to.
+	// +kubebuilder:validation:Required
+	DBClusterIdentifier *string `json:"dbClusterIdentifier"`
+	// The compute and memory capacity of the instance; for example, db.r5.large.
+	// +kubebuilder:validation:Required
+	DBInstanceClass *string `json:"dbInstanceClass"`
+	// The instance identifier. This parameter is stored as a lowercase string.
+	//
+	// Constraints:
+	//
+	//    * Must contain from 1 to 63 letters, numbers, or hyphens.
+	//
+	//    * The first character must be a letter.
+	//
+	//    * Cannot end with a hyphen or contain two consecutive hyphens.
+	//
+	// Example: mydbinstance
+	// +kubebuilder:validation:Required
+	DBInstanceIdentifier *string `json:"dbInstanceIdentifier"`
```

Comment:
Similar to others, we need to make code-generator ignore this field.

Reply:
Fixed